distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/restito/FederatedCswServer.java

```diff
@@ -0,0 +1,205 @@
+     * Provides itest verification when a specific Http call is made.
+     *
+     * @return VerifyHttp using verifyHttp method with the tub server as the parameter.
+     */
+    public VerifyHttp verifyHttp() {
+        return VerifyHttp.verifyHttp(cswStubServer);
+    }
+
+    /**
+     * Get the post being used by the stub server.
+     *
+     * @return the port being used by the stub server.
+     */
+    public int getPort() {
+        return port;
+    }
+
+    /**
+     * Set a specific capability Response. Overwrites default/previous response condition.
+     *
+     * @param resourceName name of capability file
+     * @throws IOException
+     */
+    private void setCapabilityResponse(String resourceName) throws IOException {
+        try (InputStream inputStream = getClass().getResourceAsStream(
+                "/" + resourceName)) {
+            String document = substituteTags(IOUtils.toString(inputStream));
+
+            LOGGER.debug("Capability response: \n{}", document);
+
+            whenHttp().match(get("/services/csw"))
+                    .then(ok(), contentType("text/xml"), bytesContent(document.getBytes()));
+        }
+    }
+
+    /**
+     * Set a specific query Response. Overwrites default/previous response condition.
+     *
+     * @param resourceName name of resource file
+     * @throws IOException
+     */
+    public void setQueryResponse(String resourceName) throws IOException {
+        try (InputStream inputStream = getClass().getResourceAsStream("/" + resourceName)) {
+            String document = substituteTags(IOUtils.toString(inputStream));
+
+            LOGGER.debug("Query response: \n{}", document);
+
+            whenHttp().match(post("/services/csw"), withPostBodyContaining("GetRecords"))
+                    .then(ok(), contentType("text/xml"), bytesContent(document.getBytes()));
+        }
+    }
+
+    private void setDefaultCapabilityResponse() throws IOException {
+        try (InputStream inputStream = getClass().getResourceAsStream(
+                "/csw-get-capabilities-response.xml")) {
+            String document = substituteTags(IOUtils.toString(inputStream));
+
+            LOGGER.debug("Capability response: \n{}", document);
+
+            whenHttp().match(get("/services/csw"))
+                    .then(ok(), contentType("text/xml"), bytesContent(document.getBytes()));
+        }
+    }
+
+    private void setDefaultQueryResponse() throws IOException {
+        try (InputStream inputStream = getClass().getResourceAsStream("/csw-query-response.xml")) {
+            String document = substituteTags(IOUtils.toString(inputStream));
+
+            LOGGER.debug("Query response: \n{}", document);
+
+            whenHttp().match(post("/services/csw"), withPostBodyContaining("GetRecords"))
+                    .then(ok(), contentType("text/xml"), bytesContent(document.getBytes()));
+        }
+    }
+
+    private String substituteTags(String document) {
```

Comment:
I shouldn't have written this code and used `Library.getFileContent()` instead. Can you replace those methods?